.gitignore

```diff
@@ -9,4 +9,5 @@ Gemfile.lock
 coverage/
 dump.rdb
 doc/
-docs/
\ No newline at end of file
+docs/
+zsh/_colorls
```

Comment:
I guess this `_colorls` auto-completion file will have to be removed from gitignore.

Reply:
Yes, of course!

I have removed it from the ignore file and added it to the commit instead.

And also adjusted the rakefile and gemspec accordingly.